publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategy.java

```diff
@@ -23,33 +34,36 @@ private PublicationPermissionStrategy(
     ) {
         this.userInstance = userInstance;
         this.publication = publication;
-        this.permissionStrategies = Set.of(
+        this.grantStrategies = Set.of(
             new EditorPermissionStrategy(publication, userInstance),
             new CuratorPermissionStrategy(publication, userInstance),
             new ContributorPermissionStrategy(publication, userInstance),
             new ResourceOwnerPermissionStrategy(publication, userInstance),
             new TrustedThirdPartyStrategy(publication, userInstance)
         );
+        this.restrictStrategies = Set.of(
+            new NonDegreePermissionStrategy(publication, userInstance)
+        );
     }
 
     public static PublicationPermissionStrategy create(Publication publication, UserInstance userInstance) {
         return new PublicationPermissionStrategy(publication, userInstance);
     }
 
     public boolean allowsAction(PublicationOperation permission) {
-        return !findStrategiesAllowingOperation(permission).isEmpty();
+        return !findStrategiesAllowingOperation(permission).isEmpty()
```

Comment:
how about findAllowances and findDenials/findRestrictions to make it easier to read?